Add GitHub issue templates (#8934)
crystal-lang/crystal: .github/ISSUE_TEMPLATE/feature-request.md

---
name: "\U0001F680 Feature Request"
about: I want to propose a new language feature.
labels: kind:feature
---

## Feature Request

> - Is your feature request related to a problem? Please describe cleary and concisely what is the problem.

Reviewer: ```suggestion
- Is your feature request related to a problem? Please describe clearly and concisely what is the problem.
```

- Describe the feature you would like, optionally illustrated by examples, and how it will solve the above problem.
- Describe considered alternative solutions, and the reasons why you have not proposed them as a solution here.

> - Does it induce breaking backward compatibility, if yes what is the migration path?

Reviewer: ```suggestion
- Does it break backward compatibility, if yes then what's the migration path?
```
